Sources/BraintreeShopperInsights/BTCreateCustomerSessionApi.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// The API used to create a customer session using the `CreateCustomerSession` GraphQL mutation.
+/// - Warning: This feature is in beta. It's public API may change or be removed in future releases.
+class BTCreateCustomerSessionApi {
+    
+    // MARK: - Properties
+    
+    /// Exposed for testing to get the instance of `BTAPIClient`
```

Comment:
Since apiClient is injected in the `init()` do we still need to expose the internal `apiClient`? Not entirely sure how swift handles this 😅

Reply:
ahhh i think so, this seems to be the pattern in the other client classes as well